src/lib.rs

```diff
@@ -207,6 +210,10 @@ impl Executor {
 
             // Put the thread to sleep until another task is scheduled.
             let timeout = Duration::from_millis(500);
+            tracing::trace!(
+                timeout=?timeout,
```

Comment:
I think we can use shorthand (`?timeout` instead of `timeout=?timeout`) here.
https://docs.rs/tracing/latest/tracing/#recording-fields